plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/outputconsole/GitOutputConsolePresenter.java

```diff
@@ -35,6 +35,8 @@
 
     private final List<ConsoleOutputListener> outputListeners = new ArrayList<>();
 
+    private static final String RED_COLOR = "#F62217";
```

Comment:
It will be not ok in case of printing git status, for example when we want to print red line with unstaged file we should write: print(line, ERROR). It looks strange printing error in status.